cartography/models/ontology/mapping/data/containers.py

```diff
@@ -0,0 +1,92 @@
+from cartography.models.ontology.mapping.specs import OntologyFieldMapping
```

Comment:
Do we want to include cluster / pod on here as well? 
Generally I see container management platforms structured like:

1. Kube: `cluster -> namespace -> pod -> container`
2. ECS: `cluster -> service (optional) -> task -> container`
3. Azure Container Instance: `container group -> container`
(not including Google Cloud Run because that's more like a lambda)

Overall this feels like: Cluster -> WorkloadGroup -> Workload -> Container
Azure Container Instance would only be Workload and Container

Should we have these as fields on Containers? Or should we create these as ontology nodes later and just create relationships then?